framework/src/convergence/PostprocessorConvergence.C

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PostprocessorConvergence.h"
+
+registerMooseObject("MooseApp", PostprocessorConvergence);
+
+InputParameters
+PostprocessorConvergence::validParams()
+{
+  InputParameters params = IterationCountConvergence::validParams();
+
+  params.addClassDescription(
+      "Checks convergence based on absolute and relative error of the residual.");
```

Comment:
Looks like that class description is garbage copy/paste.

I don't know about the combo idea since in this case it has parameters (e.g., max iterations).

Another design I considered was having `IterationCountConvergence` be a wrapper class that takes the name of another convergence, rather than having other convergences inherit from it. But that'd be pretty inconvenient for the user to make at least two convergences all of the time.

Reply:
So you're suggesting that we don't make it an intermediate class but a final child class? Then somehow combine them? For the combo thing, I understand how that works, but here it's a little less clear because you don't necessarily know which Convergence objects will be for which iteration type (unless we have the user supply an iteration type).

Reply:
> we don't make it an intermediate class but a final child class

I would not make it final because I would want users to be able to derive from it. But yes, I dont see it be intermediary to every other class?

> , but here it's a little less clear because you don't necessarily know which Convergence objects will be for which iteration type 

you should know when you specify "nonlinear_convergence =" in the Executioner.